app/templates/main/map.html

```diff
@@ -0,0 +1,72 @@
+<html>
+<head>
+<script src="http://maps.googleapis.com/maps/api/js"></script>
```

Comment:
If we can download the full API, we should move this logic to app/assets.py. If we want to always pull the most updated API code, we can add this to app/templates/partials/_head.html